lib/adminware/log.rb

```diff
@@ -0,0 +1,35 @@
+require 'adminware/config'
+require 'logger'
+
+module Adminware
+  class Log
+    #Configure the logger
+    def initialize
+      @config = Adminware.config
+      @logger = Logger.new(@config.logfile)
+      @logger.level = Logger::DEBUG
+      @logger.formatter = proc do |severity, datetime, progname, msg|
+      date_format = datetime.strftime("%d-%m-%Y %H:%M:%S")
```

Comment:
The (lack of) indentation confused me a bit here!

Reply:
This was me following the documentation for the logger, will change to make it clearer